template/apis/v1alpha1/README.md.tpl

```diff
@@ -0,0 +1 @@
+This is README.md for apis/v1alpha1 directory
```

Comment:
This file seems pointless haha. I'm assuming this is a placeholder for the directory so that it's created and the code-generator works properly?

I think I'd rather we just fix the code-generator problem before we merge this PR. I've assigned [this issue](https://github.com/aws-controllers-k8s/community/issues/1361) to me